tracee-ebpf/tracee/printer.go

```diff
@@ -261,6 +261,7 @@ func (p *gobEventPrinter) Init() error {
 	p.outEnc = gob.NewEncoder(p.out)
 	gob.Register(external.Event{})
 	gob.Register(external.SlimCred{})
+	gob.Register(map[string]int{})
```

Comment:
Yes, because the printer will fail to print it otherwise.
In the start I didn't put it and only received the event's value in json but not in gob.
All complicated types should be registered this way unless they are part of registered type.